ding/entry/__init__.py

```diff
@@ -20,3 +20,4 @@
 from .application_entry_trex_collect_data import trex_collecting_data, collect_episodic_demo_data_for_trex
 from .serial_entry_guided_cost import serial_pipeline_guided_cost
 from .serial_entry_gail import serial_pipeline_gail
+from .random_collect import random_collect_fn
```

Comment:
utils.py can be better

Reply:
Resolved. Also rename `random_collect_fn` to `random_collect`